source/features/dim-bots.tsx

```diff
@@ -15,13 +16,25 @@ const botSelector = [
 	'.labels [href$="label%3Abot"]'
 ].join();
 
-function init(): void {
+function tagsLoaded(): boolean {
+	return select.exists('.js-navigation-container .octicon-tag');
+}
+
+async function init(): Promise<void> {
 	for (const bot of select.all(botSelector)) {
 		// Exclude co-authored commits
 		if (select.all('a', bot.parentElement!).every(link => link.matches(botSelector))) {
 			bot.closest('.commit, .Box-row')!.classList.add('rgh-dim-bot');
 		}
 	}
+
+	if (!tagsLoaded()) {
+		await oneMutation(document.body, {subtree: true, childList: true, filter: tagsLoaded});
```

Comment:
Maybe we need some kind of timeout here? Since the promise can hang forever on commit lists with no tags.